framework/include/cppmicroservices/detail/BundleAbstractTracked.tpp

```diff
@@ -31,23 +31,25 @@ namespace detail {
 
 template<class S, class TTT, class R>
 BundleAbstractTracked<S,TTT,R>::BundleAbstractTracked(BundleContext bc)
-  : closed(false), trackingCount(0), bc(bc)
+  : closed(false)
+  , trackingCount(0)
+  , bc(bc)
```

Comment:
can we use std::move here? The parameter bc is a copy of the BundleContext. Can we move that copy to the member variable?